Jellyfin.Plugin.OpenSubtitles/OpenSubtitlesHandler/OpenSubtitlesRequestHelper.cs

```diff
@@ -10,105 +10,105 @@
-        /// </summary>
-        public static OpenSubtitlesRequestHelper? Instance { get; set; }
-
-        /// <summary>
-        /// Calculates: size + 64bit chksum of the first and last 64k (even if they overlap because the file is smaller than 128k).
-        /// </summary>
-        /// <param name="input">The input stream.</param>
-        /// <returns>The hash as Hexadecimal string.</returns>
-        public static string ComputeHash(Stream input)
+        input.Seek(-HashPos, SeekOrigin.End);
+        while (input.Read(buffer) > 0)
         {
-            const int HashLength = 8; // 64 bit hash
-            const long HashPos = 64 * 1024; // 64k
-
-            long streamsize = input.Length;
-            ulong hash = (ulong)streamsize;
+            hash += BinaryPrimitives.ReadUInt64LittleEndian(buffer);
+        }
 
-            Span<byte> buffer = stackalloc byte[HashLength];
-            while (input.Position < HashPos && input.Read(buffer) > 0)
-            {
-                hash += BinaryPrimitives.ReadUInt64LittleEndian(buffer);
-            }
+        BinaryPrimitives.WriteUInt64BigEndian(buffer, hash);
+        return Convert.ToHexString(buffer).ToLowerInvariant();
+    }
 
-            input.Seek(-HashPos, SeekOrigin.End);
-            while (input.Read(buffer) > 0)
-            {
-                hash += BinaryPrimitives.ReadUInt64LittleEndian(buffer);
-            }
+    internal async Task<(string body, Dictionary<string, string> headers, HttpStatusCode statusCode)> SendRequestAsync(
+        string url,
+        HttpMethod method,
+        object? body,
+        Dictionary<string, string> headers,
+        CancellationToken cancellationToken)
+    {
+        var client = _clientFactory.CreateClient("Default");
 
-            BinaryPrimitives.WriteUInt64BigEndian(buffer, hash);
-            return Convert.ToHexString(buffer).ToLowerInvariant();
+        HttpContent? content = null;
+        if (method != HttpMethod.Get && body is not null)
+        {
+            content = JsonContent.Create(body);
         }
 
-        internal async Task<(string body, Dictionary<string, string> headers, HttpStatusCode statusCode)> SendRequestAsync(
-            string url,
-            HttpMethod method,
-            object? body,
-            Dictionary<string, string> headers,
-            CancellationToken cancellationToken)
+        using var request = new HttpRequestMessage
         {
-            var client = _clientFactory.CreateClient("Default");
-
-            HttpContent? content = null;
-            if (method != HttpMethod.Get && body != null)
+            Method = method,
+            RequestUri = new Uri(url),
+            Content = content,
+            Headers =
             {
-                content = JsonContent.Create(body);
+                Accept = { new MediaTypeWithQualityHeaderValue("*/*") }
             }
+        };
+
+        request.Headers.Add("User-Agent", $"Jellyfin-Plugin-OpenSubtitles v{_version}");
```

Comment:
`request.Headers.UserAgent.Add("Jellyfin-Plugin-OpenSubtitles", _version);` should work

Reply:
@MBR-0001 in case you missed it :)

Reply:
I reverted to just setting it in inside constructor in https://github.com/jellyfin/jellyfin-plugin-opensubtitles/pull/115/commits/996de2db307fe099b3b504ef1bef918861382335